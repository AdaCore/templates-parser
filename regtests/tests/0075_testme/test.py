from test_support import *

run('testme', ["testme75.tmplt"])
