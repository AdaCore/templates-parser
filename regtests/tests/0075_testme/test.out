
1.
           return Fa1 (Self) = T_Rt1.Ra1 (T_Rt1.Table (Foreign)) and Fa2 (Self) = T_Rt2.Ra2 (T_Rt2.Table (Foreign)) and Fa3 (Self) = T_Rt3.Ra3 (T_Rt3.Table (Foreign));
2.
   return Fa1 (Self) = T_Rt1.Ra1 (T_Rt1.Table (Foreign))
     and Fa2 (Self) = T_Rt2.Ra2 (T_Rt2.Table (Foreign))
     and Fa3 (Self) = T_Rt3.Ra3 (T_Rt3.Table (Foreign));
